src/lib/Elf.cc

```diff
@@ -73,37 +77,38 @@ Elf::Elf(std::string path, char** imagePointer) {
    */
 
   // Seek to the byte representing the start of the header offset table.
-  uint64_t headerOffset;
-  file.read(reinterpret_cast<char*>(&headerOffset), sizeof(headerOffset));
+  // Holds the program header table's file offset in bytes.  If the file has no
+  // program header table, this member holds zero
+  uint64_t e_phoff;
+  file.read(reinterpret_cast<char*>(&e_phoff), sizeof(e_phoff));
 
   /**
-   * Starting 54th byte of the ELF Header a 16-bit value indicates
-   * the size of each entry in the ELF Program header. In the `elf64_hdr`
-   * struct this value maps to the member `Elf64_Half e_phentsize`. All
-   * header entries have the same size.
+   * Starting from the 54th byte of the ELF Header a 16-bit value indicates
+   * the the size in bytes of one entry in the
```

Comment:
"the the"

Reply:
Is this the comment you were referring to re weird formatting? It doesn't look right.

Reply:
Have managed to fix the formatting